api/cloud_accounts.go

```diff
@@ -0,0 +1,212 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
```

Comment:
curious: Is this our actual license, or just the public codebases ?

Reply:
My name is no license whatsoever 😂 - my local setup creates files with "the license header" on them which you can see below the author's name, it is built from Github's user/name/email from the person running the commands. This is a very vanilla setup I have had for over 10 years, I could share it if you are interested.

Reply:
yes please !